specs/schema.TEStribute.openapi.yaml

```diff
@@ -0,0 +1,96 @@
+openapi: "3.0.0"
+info:
+  version: 0.0.1
```

Comment:
I think we tie this to the TEStribute version. If we don't have one, let's say 0.1.0 :)